.vscode/settings.json

```diff
@@ -1,4 +1,6 @@
 {
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
-  "editor.formatOnPaste": true
+  "editor.formatOnPaste": true,
+  "files.insertFinalNewline": true
```

Comment:
This should be taken care of by editorconfig (which I forgot to add to the list of recommended extensions, sorry!)?

Reply:
(I added editorconfig to the list of recommended extensions, do we still need this?)